Code/Source/solver/mat_models.cpp

```diff
@@ -153,10 +316,12 @@ void get_fib_stress(const ComMod& com_mod, const CepMod& cep_mod, const fibStrsT
  * @param[in] ya Electrophysiology active stress.
  * @param[out] S 2nd Piola-Kirchhoff stress tensor (modified in place).
  * @param[out] Dm Material stiffness tensor (modified in place).
- * @return None, but modifies S and Dm in place.
+ * @param[out] Ja Jacobian for active strain
+ * @return None, but modifies S, Dm, and Ja in place.
  */
-void get_pk2cc(const ComMod& com_mod, const CepMod& cep_mod, const dmnType& lDmn, const Array<double>& F, const int nfd,
-    const Array<double>& fl, const double ya, Array<double>& S, Array<double>& Dm)
+template<size_t nsd>
+void _get_pk2cc(const ComMod& com_mod, const CepMod& cep_mod, const dmnType& lDmn, const EigenMatrix<nsd>& F, const int nfd,
```

Comment:
What't the `_` for ?

Reply:
@aabrown100-git You don't need a different name, can use `get_pk2cc()` for the template function, the compiler will figure out which one to use.

Reply:
Will do